homework/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "homework",
+    "version": "1.0.0",
+    "description": "Homework: JavaScript Basics",
+    "main": "index.js",
+    "scripts": {
+        "test": "mocha -bdd test.js"
+    },
+    "dependencies": {
```

Comment:
These are actually `devDependencies` as you use them in tests / to format code, not in the source code of your package.